assets/css/style.css

```diff
@@ -141,12 +130,236 @@ main {
 }
 
 .home-background {
-  background-image: url('../images/header-shapes mobile.png');
+  background-image: url('../images/backgrounds/header-shapes mobile.png');
   background-repeat: no-repeat;
   background-size: cover;
+  margin-bottom: 5em;
+  border-radius: 0 0 0 5em;
+  background-color: #fff;
+  min-height: 35em;
+}
+
+.about-background {
+  margin-bottom: 1em;
+  background-color: #fff;
+  padding: 1em;
+  padding-bottom: 10em;
+  min-height: 90vh;
 }
 
 .flex-container {
   display: flex;
   flex-direction: column;
 }
+
+.card {
+  border-radius: 1em;
+  background-color: #fff;
+  padding: 1em;
+  margin: 1em;
+  margin-bottom: 5em;
+}
+
+.work-grid {
+  display: grid;
+  grid-template:
+    '.'
+    '.'
+    '.';
+}
+
+.project-title {
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.flex-list {
+  display: inline-flex;
+  padding: 0;
+  margin-top: 0.3em;
+}
+
+.project-date-list {
+  margin-bottom: 0.5em;
+}
+
+.project-date-list > li {
+  margin-right: 0.5em;
+}
+
+.project-date-list > li > h5 {
+  /* Caption / 1 - Bolder */
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  display: contents;
+}
+
+.project-date-list > li > label {
+  /* Caption / 1 - Bolder */
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-description {
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-bottom: 0.5em;
+}
+
+.project-button-group {
+  display: flex;
+  margin-bottom: 1em;
+}
+
+.project-button-group > li {
+  margin-right: 0.3em;
+}
+
+.project-button-group > li > button {
+  /* Button / Small */
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.project-button {
+  /* Button /Normal */
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #fefefe;
+  color: #396df2;
+  border: #396df2 solid 0.1em;
+  margin-bottom: 1em;
+}
```

Comment:
I would suggest that we add more `padding-top`, `padding bottom` to the buttons similar to the Figma design

Your design
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/78251927/894bb49e-2b9b-4e89-961b-41b05f3407fb)

Figma
![image](https://github.com/chudisoft/Portfolio-Microverse/assets/78251927/9156a1c8-b72e-4d60-a1b3-8fc674f48df3)